### PR TITLE
Update current.yml

### PR DESCRIPTION
**Pull request by Anoifeiram**
Added Odin testconference in Norway :-)

### CODE REVIEW

File: _data/current.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+- name: Odin Testkonferansen Norway 2024
+  location: Oslo
+  dates: "September 26, 2024"
+  url: www.https://event.dataforeningen.no/odin/
```
Comment (ckenst) [inline _data/current.yml:4]: This looks like a typo

File: _data/current.yml [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+  twitter: N/A
```
Comment (ckenst) [inline _data/current.yml:5]: If there is no twitter, just remove the line

**Comment by ckenst** [comment]
@Anoifeiram thanks for the contribution. There are a few small things that if you can clean up, I'll merge.

File: _data/current.yml (hunk 2 above)
Comment (Anoifeiram) [reply _data/current.yml:5]: Ok, there is no twitter

File: _data/current.yml (hunk 1 above)
Comment (Anoifeiram) [reply _data/current.yml:4]: Hi, what looks like a type ?
Is it the - before mane?

File: _data/current.yml (hunk 2 above)
Comment (ckenst) [reply _data/current.yml:5]: Remove line number 5. No reason to have it in.

File: _data/current.yml (hunk 1 above)
Comment (ckenst) [reply _data/current.yml:4]: the url is: `www.https://` and that looks like a typo. Remove the www.

**Pull request closed without merging**